bot/cogs/reminders.py

```diff
@@ -153,36 +191,37 @@ async def send_reminder(self, reminder: dict, late: relativedelta = None) -> Non
                 name=f"Sorry it arrived {humanize_delta(late, max_units=2)} late!"
             )
 
+        additional_mentions = ' '.join(
+            mentionable.mention for mentionable in self.get_mentionables_from_ids(reminder["mentions"])
+        )
+
         await channel.send(
-            content=user.mention,
+            content=f"{user.mention} {additional_mentions}",
             embed=embed
         )
-        await self._delete_reminder(reminder["id"])
```

Comment:
I would have preferred you to leave this alone since my PR already fixes this #835.

Reply:
I wasn't aware there was an existing issue for this, I'll revert it.